\begin{doublespace}

\chapter{Ecological and mutation-order speciation in digital organisms}
\label{chap:ecol_mo}



\section{Introduction}

Reproductive isolation between populations
often evolves as a byproduct of independent adaptation
to new environments \citep{coy04,sch09,sob10}.
%
When these environments' selective pressures are different,
divergent selection can cause populations
to acquire different, often incompatible, alleles.
%
This divergent process can generate reproductive isolation
both in nature \citep[reviewed in][]{run05,sch09}
and in laboratory experiments \citetext{\citealp{det07,det08};
reviewed in \citealp{ric93} and \citealp{fry09}}.
%
Complete reproductive isolation due to this process
is known as `ecological speciation' \citep{sch09}.
%
On the other hand, if the environments' selective pressures
are similar or identical (parallel or uniform selection),
populations may diverge genetically
by the chance fixation of different alleles.
%
Although laboratory experiments and theoretical studies suggest that
such process may lead to `mutation-order speciation' \citep{sch09,nos11},
its effectiveness in generating reproductive isolation
compared to ecological speciation is unknown.


%Future work should
%aim to infer the roles of gene ﬂow, drift and selection in
%overall divergence and the ﬁxation of individual speciation
%genes (see Question 7) and compare their contributions
%across speciation events in different taxa and environments.
%(Marie Curie speciation group)


%It is now accepted that the
%evolution of reproductive barriers without
%spatial separation or in secondary contact zones is a plausible route
%to speciation
%(Smadja 2011)


%There is no reinforcement, no sexual selection,
%simply adaptation to resources in separate environments



The main purpose of this study is to directly compare
the strength of reproductive isolation
generated by ecological and mutation-order processes.
%
Specifically, we measure both the degree of postzygotic isolation
(i.e., hybrid inviability)
as well as the amount of genetic divergence between populations
evolved under either different environments or the same environments.
%
Because there is a higher chance of parallel evolution
when environments are similar \citep{sch09b},
we expect that postzygotic isolation and genetic divergence
under a mutation-order process will be weaker than under an ecological process.




We also examine the effect of migration
on both ecological and mutation-order processes.
%
Migration between populations increases the chance of gene flow,
which often slows genetic (and thus adaptive) divergence,
although gene flow can also promote divergence \citep{gar06,ras08}.
%
We vary the amount of migration between populations
under both ecological and mutation-order processes,
from allopatry to parapatry to sympatry.
%
We expect that migration will have a stronger negative effect
on the evolution of reproductive isolation under mutation-order scenarios
because, under uniform selection, an adaptive mutation
that arises in one population is also selectively
favored in the other \citep{sch09,nos09b,nos11}.



% IMPORTANT: Gene flow is not the same as migration
% (Whitlock and McCauley, 1999)

% I need so say somewhere here why my selective pressures are so strong,
% why didn't I vary other parameters, like mutation rate, population size
% possible reason: we want to COMPARE ecological vs. mutation-order,
%   not under which parameters it is likely to occur,
%   which has been studies elsewhere (e.g., Nosil & Flaxman, Gavrilets 2000)



We also examine how the environments' dimensionality
(i.e., number of selective pressures)
affects the strength of reproductive isolation
for both ecological and mutation-order processes.
%
In highly dimensional environments,
there are more opportunities for populations
to adapt in different ways \citep{ric93,nos09},
which may lead to stronger reproductive isolation
for both ecological and mutation-order processes.
%
High dimensionality, however, may also constrain speciation
when trade-offs among adaptive traits hinder adaptation
and therefore decrease the probability of reproductive isolation.



Finally, we examine one possible cause for differences
in hybrid fitness between ecological and mutation-order processes.
%
As populations adapt to their local environments independently,
there is no guarantee that alleles acquired in one population
will interact positively, or even neutrally, in the hybrid
with alleles acquired in the other population \citep{coy04}.
%
Negative interactions between alleles from two populations
are known as Dob\-zhan\-sky-Mul\-ler incompatibilities (DMIs),
and both ecological and mutation-order process
may cause them to form and thus lead to postzygotic isolation \citep{sch09}.
%
It is unknown, however, whether ecological or mutation-order speciation
differ in their propensity to produce DMIs.
%
Although a mutation-order process may be expected to have
fewer DMIs initially than an ecological process
because uniform selection selects for the same alleles,
such expectation diminishes the more populations diverge \citep{sch09}.



%Take-home message(s):

%Ecological speciation is more effective at creating R.I.
%- Result: Under all migration rates,
%  RI from divergent selection was stronger than mutation order
%  - Explanation: In mut-order, similar loci evolve
%    because genome arch. is constrained by the environment

%Some gene flow permits R.I. in ecological spp, but not in mutation order
%- Result: Ecological R.I. decreases with migration;
%  Mut-order R.I. present only w/ 0 migration; otherwise, no R.I.
%  - related: mut-order adaptation proceeds the same way w/ migration

%Speciation more readily in multifarious selection
%- Result: Weaker R.I. overall at low-dim than at hi-dim
%  - Explanation: Possibly because of more genome divergence;
%    in ecol., at high migration rate (0.1), no R.I at low-dim,
%    but high R.I. at hi-dim; in lo-dim, demes adapt to both environments
  
%In mut-order, even though similar loci evolve, they are not exact
%- Result: In unlinked hybridization, mut-order R.I. is as strong as ecol.

%In drift, genetic incompatibilities do occur
%- Result: In unlinked hybridization, drift R.I. is moderate

% Relevant?
%In mut-order, R.I. may depend on where hybrids find themselves
%- Result: Mut-order hybrids sometimes do better than one of the parents,
%  but worse than the other (forming a kind of "bubble" through time)



%%% METHODS %%%

% General settings
%Mass action
%Sexual reproduction
%Child size range = 1.0
%Rewards are additive.
%Copy mutations not allowed.
%Mutation rate is U = 0.1
%No other types of mutations

% Experimental design 3
%To generate the ancestral population,
%we ran 50 replicates of a \emph{world} of size 10,000
%initiated with a simple digital organism of genome length 200
%for 500,000 updates ($\sim$ 42,000 generations).
%%
%The environment rewarded the standard nine logic tasks. % cite lenski
%%
%We then randomly selected one replicate
%that could perform all nine tasks at the end of the run.



In this study, we use the software Avida to carry out our experiments.
%
Avida \citep{ofr04} is an artificial life research platform
where digital organisms evolve due to genetic variation, inheritance,
and differential reproduction (see Methods).
%
Avida has been used previously
in a wide range of ecological and evolutionary studies
\citep[e.g.,][]{len03,cho04,ele07,ost07,mis10}.
%
There are several reasons for using digital organisms to study evolution:
we can observe millions of generations in a few days,
conduct hundreds of replicate experiments,
easily manipulate genomes,
and accurately measure fitness.
%
Digital organisms in Avida are not meant to specifically mimic
the details of real biological organisms.
%
Instead, digital organisms have a unique genetic system (see Methods).
%
Despite these differences, the general principles
that make evolution possible are still the same.
%
Therefore, Avida can be used to test the generality
of evolutionary theories and hypotheses without making particular assumptions
about the details of biology.
%
Indeed, several evolutionary properties
have been found to be remarkably similar to that of biological organisms
\citep{wil02,ada06}
(e.g., the distribution of mutational effects, the types of epistasis,
and the genetic architecture of sexual organisms).
%
Digital organisms improve on simple two-locus models of speciation
because in Avida, traits are complex,
involving multiple loci and epistatic interactions among alleles \citep{len99}.


%In ecological speciation, however,
%populations evolve different phenotypes,
%and thus the combination of phenotypes
%inherited by hybrids from both parents
%cannot all function in either parental environment.
%%
%If we assume that the `average' hybrid inherits
%half its traits from one parent and half from the other,
%and if there is no genetic incompatibility between
%the genes controlling these traits,
%then hybrids from mutation-order speciation
%should do as well as their parents,
%but hybrids from ecological speciation
%should do half as well as their parents.

\section{Methods}

\subsection{Avida}



Experiments with digital organisms were carried out using Avida (ver.~2.9.0),
freely available at \url{http://avida.devosoft.org}.
%
In this section, we provide a brief overview of Avida;
for a full description, see \citet{ofr04}.
%
Avida is highly configurable, so unless otherwise noted,
the following description applies to the default settings.
%
In Avida, digital organisms are composed of
a linear sequence of instructions (akin to a haploid genome),
memory space in the form of registers and stacks,
pointers to memory locations,
and a central processing unit (CPU) that executes instructions.
%
The instruction set makes up an assembly-like programming language,
consisting of instructions for arithmetic operations,
memory manipulation (e.g., swap registers or push into a stack),
conditional execution (i.e., `if' statements), iteration (looping),
input/output operations, and allocation and copying of memory.
%
Organisms execute their instructions sequentially,
sometimes skipping instructions for conditional statements
or repeating the same instructions inside a loop;
when the last instruction is executed,
execution starts again at the first instruction.
%
By executing instructions in their genomes,
organisms are able to (1) replicate and (2) perform computational `tasks'
that affect the speed at which they replicate and thus increase fitness.



To replicate, an allocation instruction creates the memory space
required by the organism's offspring, and a copy instruction inside a loop
allows the organism to copy itself into the new memory space.
%
The copy instruction that allows organisms to replicate
has a configurable probability of making mistakes,
therefore introducing various kinds of mutations
(e.g., point mutations, indel mutations, and slip mutations).
%
By default, replication is asexual.
%
However, Avida may be configured to perform sexual replication,
in which the genomes of two asexually-produced offspring are recombined
by exchanging two randomly-sized regions of their genomes.
%
The offspring (whether clonal or two recombinants) are put into the population
in random locations, replacing whatever organisms were already there.
%
Generations are therefore overlapping, as offspring are born continuously,
replacing older individuals who are likely not their parents.



In addition to replication, genomic instructions allow organisms
to acquire 32-bit input values and use them to perform computational tasks.
%
Tasks are boolean operations, such as NOT, AND, and OR,
and are applied to input values bit by bit.
%
For example, if input values were 8 bits,
the operation 10011101 AND 11101011 would produce 10001001
because 0 AND 0 is 0, 0 AND 1 is 0, 1 AND 0 is 0, and 1 AND 1 is 1.
%
In Avida, however, there is no AND operation nor any other boolean operation
except for NAND, from which all other boolean operations may be built,
a property of NAND known as `functional completeness' in boolean algebra.
%
For example, P AND Q, where P and Q are input values,
is equivalent to (P NAND Q) NAND (P NAND Q),
and P OR Q is equivalent to (P NAND P) NAND (Q NAND Q).
%
Because tasks are made up of the same building blocks (NANDs),
evolved tasks may share the same pieces of code
and therefore may not be completely independent.



When an organism performs a task, the organism's `merit' is increased
by a specific amount, specified in a configuration file, for that task.
%
The merit of an organism is a unitless value used by Avida
to determine the number of instructions an organism may execute each time step.
%
If two organisms had the same merit,
they would execute the same number of instructions at each time step;
however, if one organism had twice the merit as another,
the first organism would execute twice the number of instructions
compared to the second in a single time step.
%
Thus, an organism with twice the merit as another,
would replicate twice as fast.
%
Organisms initially inherit the merit of their parents;
otherwise, new organisms would be at a disadvantage
compared to the rest of the population.
%
The default environment rewards for nine binary (i.e., two-input) tasks,
but the environment can be configured to reward for an additional 68
three-input tasks.



Adaptation in Avida occurs naturally (i.e., it is not simulated),
as a result of the three ingredients required for natural selection:
inheritance, variation, and differential reproduction.
%
Inheritance comes from replication (sexual or asexual),
variation comes from mutation and recombination,
and differential reproduction comes from their rate of replication
(determined by their replication code and performance of tasks).
%
The ability to perform tasks evolves as organisms with the right mutations
replicate faster than others and therefore take over the population.
%
There are many ways in which to perform any one task,
and independently evolved organisms often evolve the same task
in different ways and with different degrees of efficiency.



%ecological speciation happens b/c traits under
%divergent natural selection affect reproductive compatibility
%- similarly, in avida, traits determine reproductive speed,
%  when hybrids get a mixture of traits so that they perform
%  fewer useful traits in a certain environment,
%  their reproductive speed decreases, so it does
%  affect reproductive compatibility



\subsection{Experimental design}



The Avida configuration files used to run our experiments
are available in the Dryad Data Repository.
%
To generate the ancestral population,
we initiated an empty `world' under the default nine-task environment
with an organism that could replicate but could not perform any tasks.
%
We then let this population evolve
for 500,000 updates ($\sim$~42,000 generations).
%
An `update' is a measurement of time in Avida, increasing by one each time
organisms execute 60 instructions (on average).
%
For the evolution of the ancestral population,
we set the world size to 10,000 individuals.
%
The length of the genome was set to 200 instructions,
and to ensure homologous recombination
during sexual reproduction, the genome length was fixed.
%
The mutation rate was set to 0.1 mutations per genome per generation.



We then setup four treatments (described below), which we call
`drift,' `ecological,' `mu\-ta\-tion-or\-der 1,' and `mu\-ta\-tion-or\-der 2.'
%
For each treatment,
the world size was set to 2,000 individuals
and divided into two demes, each of size 1,000.
%
In the drift treatment, both demes' environments
were the same as the ancestral (environment `A').
%
For each remaining treatments,
we setup two subtreatments (described below):
low dimensionality and high dimensionality.
%
For this study, dimensionality refers to the number of tasks
for which that environment rewards organisms for performing such tasks.
%
In the ecological treatment, the demes' environments
were different from each other and different from the ancestral
(environments `1L' and `2L' for low dimensionality
and environments `1H' and `2H' for high dimensionality).
%
In the mutation-order 1 treatment,
the demes' environments were the same as each other
but different from the ancestral
(environment `1L' for low dimensionality
and environment `1H' for high dimensionality).
%
Similarly, in the mutation-order 2 treatment,
the demes' environments were the same as each other
but different from the ancestral
(environment `2L' for low dimensionality
and environment `2H' for high dimensionality).



The tasks rewarded for each environment
are shown in Table \ref{table:env}
and remained as such for the rest of this study.
%
The number of tasks for the low and high environments
were chosen as two extremes: two tasks for low dimensionality
and the maximum of 34 (i.e., 68 possible tasks divided randomly
into two demes) for high dimensionality.
%
The specific tasks for environments 1L and 2L were chosen at random
from tasks that were known to evolve within 10,000 updates in preliminary runs.
%
Note that environments 1L and 2L share no tasks;
similarly, environments 1H and 2H share no tasks.

\begin{table}
\centering
\begin{tabular}{ccccc}
\hline
\multicolumn{5}{c}{Environment} \\
\hline
A & 1L & 2L & 1H & 2H \\
\hline
NOT&3BO&3BH&3AC&3AA \\
NAND&3CI&3BJ&3AD&3AB \\
AND&&&3AE&3AF \\
ORN&&&3AG&3AH \\
OR&&&3AL&3AI \\
ANDN&&&3AO&3AJ \\
NOR&&&3AP&3AK \\
XOR&&&3AQ&3AM \\
EQU&&&3AR&3AN \\
&&&3AS&3AT \\
&&&3AU&3AV \\
&&&3AW&3AZ \\
&&&3AX&3BE \\
&&&3AY&3BG \\
&&&3BA&3BH \\
&&&3BB&3BI \\
&&&3BC&3BJ \\
&&&3BD&3BL \\
&&&3BF&3BM \\
&&&3BK&3BN \\
&&&3BO&3BQ \\
&&&3BP&3BS \\
&&&3BR&3BX \\
&&&3BT&3BY \\
&&&3BU&3BZ \\
&&&3BV&3CA \\
&&&3BW&3CB \\
&&&3CF&3CC \\
&&&3CH&3CD \\
&&&3CI&3CE \\
&&&3CJ&3CG \\
&&&3CL&3CK \\
&&&3CM&3CN \\
&&&3CO&3CP \\
\hline
\end{tabular}
\caption{Tasks rewarded in each environment.}
\label{table:env}
\end{table}



Each treatment was replicated 20 times
with a different random sample of 2,000 organisms (1,000 per deme)
from the ancestral population.
%
Successive random samples were reused for each treatment,
so that the genotypes in replicate $n$ of a treatment
were the same as the genotypes in replicate $n$ of another treatment.
%
We ran each replicate for 10,000 updates ($\sim$~850 generations).
%
For each run, the entire population of organisms
was saved every 100 updates ($\sim$~8.5 generations).
%
To examine the effect of gene flow,
each replicate was run under eight migration rates
(for the entire length of the run):
0.0 (allopatry), 0.00001, 0.0001, 0.001, 0.01, 0.05, 0.1, and 0.5 (sympatry).
%
Migration rate is the probability of an offspring
being born in a deme different than its parents;
the migrating offspring was placed in a random
location in the other deme (i.e., there were no hybrid zones)
but the parents remained in their own demes.
%
In all, there were 1,120 runs.



We measured the overall strength of selection in each environment
(1L, 2L, 1H, and 2H) at the end of each `ecological' replicate run.
%
First, for every organism in a population that could perform at least one task,
we counted the number of tasks it could perform
and calculated its fitness relative to the mean.
%
We then used linear regression on the relationship between
the number of tasks an organism could perform and its relative fitness.
%
The slope of this line is the strength of selection \citep{con04}.
%
We report these results here, as they are part of the environment
in which populations evolved.
%
For environments 1L and 2L, the mean strengths of selection were
0.3440 (0.3375--0.3562, 95\% bootstrap C.I.) and 0.3353 (0.3341--0.3366),
respectively.
%
For environments 1H and 2H, the mean strengths of selection were
0.2045 (0.1852--0.2248) and 0.1899 (0.1762--0.2041), respectively.
%
We discuss the strength of selection in the Discussion.



\subsection{Postzygotic reproductive isolation}

In this study,
we focus on the evolution of postzygotic reproductive isolation.
%
To measure the strength of postzygotic isolation for each treatment,
we first selected 1,000 random pairs of organisms (one from each deme)
and created one hybrid per pair at the end of each replicate run.
%
We then calculated the fitness of each hybrid
as the mean fitness relative to each parent.
%
Finally, we compared the mean hybrid fitnesses for each treatment---%
the lower this fitness, the stronger the isolation.
%
Note that hybrids were created
after the experiments were finished;
no hybrids were put back into the population.



Two types of hybridizations were performed for creating hybrids
after the populations had evolved.
%
The first followed the method used in Avida for sexual reproduction
(and the way in which all our experimental populations experienced):
a randomly-sized genomic region starting at a random locus was chosen
(both random numbers came from a uniform distribution),
and two recombinant offspring were created by exchanging
the genetic region of one parent with the other (two-point crossover).
%
We randomly chose one of the two offspring as the hybrid.
%
We also performed a more fine-scaled hybridization method,
where each locus of a hybrid had the same probability (0.5)
of it coming from either one or the other parent.
%
This method effectively increased the number of crossover points up to 200
and the number of regions that can be exchanged up to 100.
%
We used this multiple-point crossover method
to break apart coadapted gene complexes,
following the same logic that researchers
use when carrying out parental backcrosses
or intercrosses between hybrids \citep[e.g.,][]{li97,bur99}.
%
Multiple crossover points can expose
incompatible gene complexes between species,
revealing patterns of divergence
that would be difficult to detect with
recombination at only two crossover points.



\subsection{Genetic divergence}



To quantify the homogenizing effects of gene flow,
we calculated the genetic divergence
between each replicate pair of demes
under 0.0 and 0.01 migration for each treatment.
%
Genetic divergence was measured as the fixation index
$F_{\mathrm{ST}} = 1 - H_{\mathrm{S}} / H_{\mathrm{T}}$,
where $H_{\mathrm{S}}$ is the mean heterozygosity of each deme
and $H_{\mathrm{T}}$ is the heterozygosity of both demes
treated as one population \citep[p. 118]{har97}.
%
The heterozygosity of a deme
is the mean heterozygosity at all loci.
%
The heterozygosity at a locus is $H = 1 - \sum_{i = 1}^{n} x_i^{2}$,
where $n$ is the number of alleles segregating at that locus
and $x_i$ is the frequency of the $i$th allele \citep[p. 15]{gil04}.
%
$F_{\mathrm{ST}}$ values between 0 and 0.05
would indicate little or no genetic divergence
between two demes \citep[p. 118]{har97}.
%
We expect that under no migration
$F_{\mathrm{ST}}$ values will be significantly higher
than those under the 0.01 migration rate.
%
Significance among treatments was determined by comparing their
95\% confidence intervals of the mean $F_{\mathrm{ST}}$.
%
Each confidence interval was estimated by calculating 10,000 means
of random samples (with replacement) of the $F_{\mathrm{ST}}$ values
from the 20 replicates
(i.e., each sample contained 20 $F_{\mathrm{ST}}$ values).
%
The interval between 2.5\% and 97.5\% of means defined the confidence interval.



To test whether gene flow causes the same mutations%
---specifically those involved in performing a task---%
to fix under mutation-order speciation,
we carried out a two-step process to identify and map such mutations.
%
First, to determine whether the fixed mutations in a deme
were necessary to perform a task,
we reverted each locus, one by one, of the deme's consensus sequence
to the ancestral state.
%
If any reversion eliminated the ability to perform a task,
then the allele at that locus must be important for that task.
%
We ignored loci in which a reversion caused
complete inviability of the organism,
as these loci were involved in more than just task performance.
%
Second, we aligned the consensus sequences of each pair of demes
under 0.0 and 0.01 migration and highlighted the mutations we found above.



\subsection{Hybrid phenotypes}



To examine a possible cause for differences
in hybrid fitness between ecological and mutation-order processes,
we counted the number of times that hybrids had low fitness
due to Dob\-zhan\-sky-Mul\-ler incompatibilities (DMIs).
%
Under two-point crossover recombination,
a hybrid is made up of two parental components,
which we call C$_{1}$ and C$_{2}$.
%
If C$_{1}$ or C$_{2}$ contains the instructions to perform a task
but the full hybrid cannot perform that task,
C$_{1}$ and C$_{2}$ must interfere with one another through at least one DMI.
%
To determine whether a hybrid had low fitness due to DMIs,
we constructed two genotypes by making two copies of the hybrid,
where we replaced C$_{2}$ in the first copy, and C$_{1}$ in the second copy,
with the corresponding ancestral genetic region.
%
In this way, we constructed two `component' genotypes,
where each parental component was isolated
in the genetic background of the ancestor.
%
We then determined the tasks that these component genotypes
as well as the original hybrid could perform.
%
If either component genotype could perform a task
but the hybrid could not, then at least one DMI was present.
%
We performed this analysis on 1,000 hybrids per replicate
in both the ecological and mutation-order treatments
under no migration, low-dimensionality, and two-point crossover recombination.



Because in this system an organism's fitness is largely determined
by the number and type of tasks it can perform (i.e., its phenotype),
we identified the tasks that could be performed by each hybrid
for both ecological and mutation-order processes.
%
Note that this analysis is independent of the environment
because an organism may have the ability to perform a task
even if the environment does not reward for it.
%
For simplicity, we focused only on
the no migration, low dimensionality set of treatments
that were hybridized with a two-point crossover.
%
For the ecological treatment,
hybrids were categorized by the number tasks they could perform:
(`\mbox{0-0}')~no tasks in either environment,
(`\mbox{1-0}')~one task in one environment but none in the other,
(`\mbox{1-1}')~one task in each environment,
(`\mbox{2-0}')~two tasks in one environment but none in the other,
(`\mbox{2-1}')~two tasks in one environment and one in the other,
and (`\mbox{2-2}')~two tasks in both environments.
%
For the mutation-order treatment,
hybrids were categorized by the tasks they could perform:
(`None')~no tasks,
(`1')~task~1,
(`2')~task~2,
and (`1 and 2')~both tasks.
%
For those hybrids that could perform both tasks
in the mutation-order treatment,
we determined the tasks that each hybrid's parental components could perform.
%
We categorized these parental components as
(`0,0')~no parental component performs any task,
(`1,0')~one parental component performs one task but the other none,
(`1,1')~each parental component performs a different task,
and (`2,*')~at least one parental component performs both tasks.
%
This analysis will reveal the reason, at the phenotypic level,
for differences in hybrid performance between ecological
and mutation-order processes.
%
Four replicates from the ecological treatment,
one replicate from the mutation-order 1 treatment,
and six replicates from the mutation-order 2 treatment
were removed from the analysis above.
%
In the ecological treatment, the removed replicates contained parents
that could fortuitously perform a task of the other environment
(even though there was no selective pressure for that task),
and thus it would be unclear from which parent the task was inherited
by the hybrids.
%
In the mutation-order treatments, the removed replicates contained parents
that could not perform both tasks and, therefore, was the reason
that some of the hybrids were unfit.



\section{Results}

%Take-home message(s):

%Ecological speciation is more effective at creating R.I.
%- Result: Under all migration rates,
%  RI from divergent selection was stronger than mutation order
%  - Explanation: In mut-order, similar loci evolve
%    because genome arch. is constrained by the environment

%Some gene flow permits R.I. in ecological spp, but not in mutation order
%- Result: Ecological R.I. decreases with migration;
%  Mut-order R.I. present only w/ 0 migration; otherwise, no R.I.
%  - related: mut-order adaptation proceeds the same way w/ migration

%Speciation more readily in multifarious selection
%- Result: Weaker R.I. overall at low-dim than at hi-dim
%  - Explanation: Possibly because of more genome divergence;
%    in ecol., at high migration rate (0.1), no R.I at low-dim,
%    but high R.I. at hi-dim; in lo-dim, demes adapt to both environments
  
%In mut-order, even though similar loci evolve, they are not exact
%- Result: In unlinked hybridization, mut-order R.I. is as strong as ecol.

%In drift, genetic incompatibilities do occur
%- Result: In unlinked hybridization, drift R.I. is moderate

% Relevant?
%In mut-order, R.I. may depend on where hybrids find themselves
%- Result: Mut-order hybrids sometimes do better than one of the parents,
%  but worse than the other (forming a kind of "bubble" through time)


\subsection{Postzygotic reproductive isolation}

% I want readers to quickly tie the results w/ their expectations,
% so in the Intro I must have explained what each factor
% is expected to do to R.I. and its significance

% Readers need to know:
% - if hybrid fitness is lower in one treatment vs. another
%   then R.I. is stronger in that treatment
When hybrids between the evolved demes
were created by recombining a single genetic region
(`two-point crossover'), % possible figure explaining hybridization types
reproductive isolation between demes
that adapted to different environments (ecological treatment)
was considerably stronger than
reproductive isolation between demes
that adapted to the same environment (mutation-order treatment)
(Figs. \ref{hybrid_fitness}A and \ref{hybrid_fitness}B).
%
\begin{figure}
\centering
\includegraphics[width=0.95\linewidth]{hybrid_fitness.pdf}
\caption{Mean fitness of hybrids (y-axis)
  between populations evolved under various migration rates (x-axis)
  and different treatments (markers and lines):
  the ancestral environment (`Genetic drift'),
  different new environments (`Ecological'),
  or the same new environment (`Mutation-order 1' and `Mutation-order 2').
  At low dimensionality (A and C),
  the new environments rewarded only two tasks;
  at high dimensionality (B and D),
  the new environments rewarded 34 tasks.
  With two-point crossover (A and B),
  only one region was exchanged when creating hybrids;
  with multiple-point crossover (C and D),
  up to 100 regions were exchanged.}
\label{hybrid_fitness}
\end{figure}
%
With no migration, for instance,
reproductive isolation in the ecological treatment
was more than twice as strong than in the mutation-order treatment.
%
%Reproductive isolation in the Ecological treatment, however,
%was never `complete':
%the mean hybrid fitness was never below about 0.5.
%
% Readers need to know:
% - Drift is expected to not show much R.I.
There was no reproductive isolation between demes
evolving neutrally in the ancestral environment (drift treatment):
the mean hybrid fitness was $>$~0.99 at all migration rates.



% Readers need to know:
% - Mutation-order treatment is more sensitive to gene flow,
%   Ecological treatment can withstand some gene flow and still have R.I.
Reproductive isolation in the mutation-order treatment
was more sensitive to gene flow than in the ecological treatment
(Figs. \ref{hybrid_fitness}A and \ref{hybrid_fitness}B).
%
At the 0.01 migration rate, for instance,
the mean hybrid fitness in the mutation-order treatment was $>$~0.98,
but in the ecological treatment reproductive isolation
was almost as strong as without migration.



% Readers need to know:
% - High dimensinoality is expected to form stronger R.I.
When the environment rewarded for many tasks (high dimensionality),
reproductive isolation was often stronger than
when the environment rewarded for only two tasks (low dimensionality)
(compare Figs. \ref{hybrid_fitness}A and \ref{hybrid_fitness}B,
Figs. \ref{hybrid_fitness}C and \ref{hybrid_fitness}D).
%
This pattern was most evident in the ecological treatment,
even at moderately high migration rates;
for example, the mean hybrid fitness in the ecological treatment
at 0.1 migration was 0.97 under low dimensionality %(0.9556156,0.9813019) 95%CI
but only 0.74 under high dimensionality. % (0.7196453, 0.7576181) 95% CI
%
In the mutation-order treatments, however,
reproductive isolation under high dimensionality at migration rates $>$ 0
was not always stronger than under low dimensionality,
showing again that mutation-order was sensitive to gene flow.



% Readers need to know:
% - Fine recombination can test for how well nearby alleles work together
% - Strong R.I. in this case means divergence
When hybrids between the evolved demes
were created by recombining up to 100 genetic regions
(`multiple-point crossover'),
reproductive isolation in the ecological and mutation-order treatments
was stronger (Figs. \ref{hybrid_fitness}C and \ref{hybrid_fitness}D).
%
Note that recombination with multiple crossover points was used only
to create hybrids for the calculation of postzygotic isolation;
all populations were evolved under two-point crossover recombination.
%
The mean hybrid fitness with multiple-point crossover
was significantly lower than that with two-point crossover,
dropping 33\% and 48\% in the ecological treatment
for low and high dimensionality (respectively)
and 53\% and 43\% in the mutation-order treatments.
%
The difference in strengths of reproductive isolation
between ecological and mutation-order treatments
was now smaller than that with two-point crossover.
%
Reproductive isolation in the mutation-order treatment
remained more sensitive to gene flow than in the ecological treatment.
%
% Readers need to know:
% - Divergence can build by genetic drift alone
Reproductive isolation in the genetic drift treatment with little migration
was significantly greater than with two-point crossover.
%but at migration rate $>$ 0.01
%mean hybrid fitness was $>$ 0.95.
%
% Readers need to know:
% - Mutation-order and Ecological treatments
%   are expected to form stronger R.I. than Genetic drift
% - How sensitive is genetic drift to gene flow?
Interestingly, reproductive isolation in the genetic drift treatment
with 0.00001 migration and high dimensionality
was even greater than in the mutation-order 2 treatment.


% I DON'T HAVE A CONTROL FOR FREE RECOMBINATION;
% THUS, RI SEEN IN DRIFT MAY NOT BE REAL


\subsection{Genetic divergence}

The genetic divergence under no migration was significantly higher
than that under 0.01 migration for all treatments (Table \ref{tbl_fst}),
demonstrating that gene flow between populations had a homogenizing effect.
%
\begin{table}
\centering
\begin{tabular}{llcc}
\cline{3-4}
  & & \multicolumn{2}{c}{Migration rate} \\
\hline
Dimensionality & Treatment & 0.0 & 0.01 \\
\hline
--  & Drift            & 0.3136$^{a}$ & 0.0187$^{b}$ \\
\hline
Low & Ecological       & 0.3173$^{a}$ & 0.1387$^{c}$ \\
Low & Mutation-order 1 & 0.3096$^{a}$ & 0.0186$^{b}$ \\
Low & Mutation-order 2 & 0.3002$^{a}$ & 0.0216$^{b}$ \\
\hline
High & Ecological       & 0.4187$^{d}$ & 0.2680$^{e}$ \\
High & Mutation-order 1 & 0.4221$^{d}$ & 0.0304$^{b}$ \\
High & Mutation-order 2 & 0.3758$^{f}$ & 0.0199$^{b}$ \\
\hline
\multicolumn{4}{p{4in}}{\small\emph{Note:} Shared superscript letters indicate
  that those values are not significantly different
  (95\% bootstrap confidence interval).}
\end{tabular}
\caption{Genetic divergence ($F_{\mathrm{ST}}$)
  between demes for each treatment.}
\label{tbl_fst}
\end{table}
%
Under 0.01 migration, the mutation-order treatments had little genetic
divergence ($F_{\mathrm{ST}} <$ 0.05), which was significantly lower than the
ecological treatments, suggesting that the mutation-order treatments were more
sensitive to gene flow than the ecological treatments.
%
Interestingly, the drift treatment under no migration showed high levels of
genetic divergence, as high as the ecological and mutation-order treatments for
low dimensionality.
%
Under no migration, the genetic divergence for each treatment for high
dimensionality was significantly higher than those for low dimensionality.
%
Under 0.01 migration, the genetic divergence for the ecological treatment for
high dimensionality was significantly higher than that for low dimensionality,
but for the mutation-order treatments there was no difference in the genetic
divergence between low and high dimensionalities.



In the mutation-order treatment,
the derived alleles necessary to perform
each task in a low dimensional environment
(whether it was Environment 1 or 2)
were generally clustered together
(Fig. \ref{task_muts}).
%
Without migration,
the derived alleles of one deme for a particular task
were nearby the derived alleles of the other deme for the same task.
%
With 0.01 migration, however,
the derived alleles of one deme for a particular task
were in almost the exact locations
as the derived alleles of the other deme for the same task.
%
This result suggests that the reason that reproductive isolation
is mostly absent under a mutation-order process with gene flow
is that the key mutations that allow organisms to perform tasks are the same
(i.e., no genetic divergence for important mutations).

\begin{figure}
\centering
\includegraphics[height=5in]{task-muts.pdf}
\caption{Consensus sequences of the first five evolved replicate pairs of demes
  in the mutation-order~1 treatment under no migration (top)
  and 0.01 migration (bottom).
  %
  Sequences were 200 instructions in length,
  but only the loci that differed among
  each set of five replicates are shown.
  %
  Derived alleles involved in performing a task are highlighted
  (black highlight~=~task~1, gray highlight~=~task~2, bold font~=~both).}
\label{task_muts}
\end{figure}

% We cannot estimate effective gene flow because we violate
% several assumptions (Whitlock and MacCauley 1999)

% Possible discussion point:
% It is interesting that Fst for mut-order and selection are about the same,
% but hybridization indicates different levels of differentiation;
% hybridization shows "functional" differentiation, whereas
% Fst shows genetic differentiation;
% this discrepancy in both forms of differentiation
% shows that gene complexes (tight modules) formed,
% but were different for different mut-order populations
%


\subsection{Hybrid phenotypes}

In the ecological treatment,
we found that each replicate had, on average,
268.3 hybrids (218.0--315.8, 95\% bootstrap mean C.I.) of 1,000
that contained at least one DMI between their parental components.
%
In the mutation-order treatments,
this quantity was 77.1 (45.8--111.2) and 128 (83.5--176.5) of 1,000.
%
Therefore, populations that adapted to different environments
accumulated more DMIs than populations that adapted to similar environments.



Because hybrids, on average, inherit half the genome of each parent,
we expected that hybrids, on average, would inherit half the tasks
from each parent (here we focused on the treatments without migration,
low dimensionality, and two-point crossover).
%
In the ecological treatment, we found that hybrids were more likely to perform
zero, one, or two tasks from one parent and none from the other.
(Figure \ref{hybrid_counts}A).
%
Less than 10\% of hybrids were able to perform all four tasks.
%
For the mutation-order treatments,
most hybrids could perform both tasks
(Figure \ref{hybrid_counts}B and \ref{hybrid_counts}C),
but because the parents could also perform both tasks,
this information alone did not tell whether hybrids inherited
one task from each parent or some other combination.
%
When we analyzed those hybrids that could perform both tasks,
we found that most inherited both tasks from just one parent
(Figure \ref{fit_hybrid_components}),
although for mutation-order 2
the difference between those
that performed one task from each parent
and those that performed both tasks from one parent
was not significant.
%
Surprisingly, for the mutation-order 1 treatment
there were many hybrids that were fit
even though their parental components
could perform no tasks or just one task
(Figure \ref{fit_hybrid_components}A).

\begin{figure}
\centering
\includegraphics[width=0.95\linewidth]{hybrid_counts.pdf}
\caption{Number of hybrids able to perform certain tasks (see Methods) for
  the (A) ecological, (B) mutation-order-1, and (C) mutation-order-2 treatments
  under no migration, low dimensionality, and two-point crossover.
  %
  Each point is a hybrid count (out of 1,000) for a single replicate.
  %
  Counts from the same replicate are connected by gray lines.
  %
  The mean hybrid count per category among replicates
  is indicated by a horizontal bar.
  %
  Non-significant differences between the mean hybrid counts
  share the same letter above (or below) the points in each category.}
\label{hybrid_counts}
\end{figure}

\begin{figure}
\centering
\includegraphics[width=0.95\linewidth]{fit_hybrid_components.pdf}
\caption{Number of fit hybrids (i.e., can perform two tasks)
  whose parental components can perform certain tasks (see Methods)
  for the (A) mutation-order 1 and (B) mutation-order 2 treatments
  under no migration, low dimensionality, and two-point crossover.
  %
  Each point is a hybrid count for a single replicate.
  %
  Counts from the same replicate are connected by gray lines.
  %
  The mean hybrid count per category among replicates
  is indicated by a horizontal bar.
  %
  Non-significant differences between the mean hybrid counts
  share the same letter above the points in each category.}
\label{fit_hybrid_components}
\end{figure}





% Make the following (fine recombination)
% be part of a prediction derived
% from the fact that tasks are "modularized"


\section{Discussion}

% My results in relation to others' (supported, not supported)

% every argument/paragraph must end with a conclusion

% a conclusion may be:
% - summary of issue i'm discussing, including my results
% - a recommendation
% - a speculation that may lead to a new hypothesis
% - a new principle
% - a statement saying that there is no conclusion

% develop my ideas from my results,
% and relate them to the general scientific discipline

% present limitations of my study

% Discussion discusses relevance, usefulness, and limitations
% of experiments and results, and possible new research

% Sort my arguments (always derived from results) by most important to least
% Cat. 1: directly relevant to original hypothesis
% Cat. 2: also relevant to hypothesis, but less conclusive
% Cat. 3: not directly relevant to hypothesis, but interesting
% Cat. 4: not as interesting (thus, eliminate altogether)

% Put more important arguments first and give them more space than others
% First paragraph: most important point

% Don't make premature generalizations based on data

% Arguments should be based on the best, most-current scientific 'authority'

% Raise possible new hypothesis, or ideas, but based on evidence

% Don't repeat Results, point to Figures/Tables if needed


In this study, we used experimental evolution of digital organisms
to compare the strength of postzygotic reproductive isolation
generated by ecological and mutation-order processes.
%
We assessed the strength of postzygotic isolation
by measuring the mean hybrid fitness relative to each parent
in its native environment.
%
We found that, using a two-point crossover recombination method,
the mean hybrid fitness was around 55\% under ecological divergence
but around 83\% under a mutation-order process.
%
Other studies have also found that the mean hybrid fitness
is lower under divergent selection than under parallel selection.
%
\citet{det07} found that the mean relative fitness of hybrids
between yeast populations evolved in different environments
(high-salinity and low-glucose) was around 87\%,
but hybrids from populations evolved
under the same environmental conditions were as fit as their parents.
% I calculated the 87% as mean of (1.22/1.44) and (1/1.12) from their Fig. 2
%
Similar patterns were found in a filamentous fungus by \citet{det08},
although in one of the parental environments
hybrids between populations under divergent selection
performed better than hybrids under parallel selection.
%
Along with these studies, our study supports
the view that ecological divergence causes
stronger reproductive isolation than a mutation-order process.



It has been suggested that gene flow during speciation
may be common \citetext{\citealt{coy04}, p. 112; \citealt{nos08}}.
which requires that genetic divergence with gene flow be possible.
%
We found that a migration rate of 1\% was not enough to prevent
genetic divergence under an ecological process (Table \ref{tbl_fst}).
%
This finding supports the notion that it is possible for populations
under divergent selection in the face of gene flow to continue to diverge.
%
Under a mutation-order process, however, a migration rate of 1\%
was enough to prevent genetic divergence,
which suggests that mutation-order speciation
is more sensitive to gene flow than ecological speciation.
%
\cite{nos11} also found in their computer simulations
that genetic divergence under a mutation-order process
did not occur $>$~1\% gene flow.
%
Therefore, different populations under similar selective pressures
probably require almost complete isolation for divergence to occur.


%restriction of migration eases the 
%conditions for speciation (Felsenstein)

%There  is  a  continual  conflict 
%between  selection,  which  increases  the  as- 
%sociation  between  isolating  mechanisms, 
%and  recombination,  which  reduces  it.  Re- 
%stricting  this  recombination  can  only  im- 
%prove  chances  for  speciation.
%(Felsenstein)
%  - seems like this was in the sence of breaking
%    the assortative mating locus and adaptations
%    rather than breaking up adaptations themselves
%    and thereby producing postzygotic isolation

%gene flow constantly introduces the wrong combination of genes
%into a local population and recombination breaks-up associations
%between genes under selection and those causing reproductive isolation
%(Nosil and Feder)

%It would be interesting to see how strength of selection
%and mutation rate affect results.
%- my current strength of selection is s = 1 for each task evolved (quite high)
%- but this strength seems ok for bacterial populations (Lenski's)

%Theoretical and empirical
%studies have argued that an intermediate level of gene
%ﬂow is optimal for adaptive divergence
%(Smadja 2011)
%- I don't see this

% People test the constraining effect of gene flow
% by measuring degree of divergence under allopatry and parapatry,
% which I have done here and found that the more migration,
% the lower the divergence (from Garant et al 2006)


% Even though hybrids may be unfit, reinforcement does not occur
% because there is no mate choice mechanism implemented.



% Future work, Avida could be used to estimate relationship b/w
% gene flow and migration rates and selection

% Also, I have to explain that with too much gene flow,
% adaptation doesn't even happen, which is why RI doesn't happen

% Traits causing reproductive isolation:
% any trait can cause RI because traits determine fitness





% 9/26/12
%Dimensionality:

%Nosil et al. TREE paper (say what I said in Results
%  and point to literature related to that)
%In Avida, first allows us to directly manipulate dimensionality;
%  people do indirect things; nice opportunity how dimenisonality might
%  affect evolutionary dymics
%
%
%Amount of RI and Fst and "decoupled"; explain what I said to Luke;
%  (look also at Rice & Hostert, Luke's links in Skype)
%
%DMI point should be part of the "Wheraas a migration" paragraph
%
%Nosil & Harmon (book chapter); "< 10\% ecol. hybrids" is a minor point;
%  keep in the same paragraph as "ecol RI > MO RI"



Reproductive isolation between populations evolving in high dimensional
environments has been predicted and observed to be stronger than in single or
low dimensional environments \citep{ric93,nos09,nos09b}.
%
In \emph{Timema} walking-stick insects, for example, reproductive isolation
showed a positive correlation with environmental dimensionality
\citep{nos08b,nos09b}; further examples are reviewed in \citet{nos09}.
%
Most empirical studies, however, rely on incomplete measures of dimensionality
(imagine the difficulty in accounting for all selective pressures in the field).
%
In this study, we were able to control precisely the number of selective
pressures for the low and high dimensionality treatments.
%
We found that under an ecological process, reproductive isolation was stronger
between populations in high dimensional environments than in low dimensional
environments.
%
Under a mutation-order process, however, this pattern held only when no
migration occurred between populations, but when gene flow was allowed this
pattern went away.
%
Our results support previous findings that dimensionality matters for
ecological speciation but suggests that for mutation-order speciation with gene
flow, environmental dimensionality may not be as important.



%Amount of RI and Fst and "decoupled"; explain what I said to Luke;
%  (look also at Rice & Hostert, Luke's links in Skype)
This conclusion was also supported by our measurements of genetic divergence:
there was no difference in genetic divergence between low and high
dimensionality for the mutation-order treatments under some gene flow.
%
For the ecological treatment, however, the genetic divergence in high
dimensionality was higher than in low dimensionality and higher than the
mutation-order treatments, again showing that mutation-order treatments were
more sensitive to gene flow.
%
Interestingly, under no migration the drift treatment (where mutations fixed
neutrally) was as high as the ecological and mutation-order treatments under
low dimensionality, suggesting that most of the divergence in the ecological
and mutation-order treatments was actually the result of neutral fixations and
few adaptive mutations.
%
Another result to note is that under no migration, the ecological and
mutation-order treatments had about the same level of genetic divergence, which
is closer to our result with multiple-point crossover than two-point crossover,
suggesting that in some cases the amount of postzygotic reproductive isolation
and genetic divergence are decoupled.



Traits that are physically modular
are hardly broken apart by recombination,
hiding genetic incompatibilities
that may have formed between populations.
%
To determine whether genetic incompatibilities
had formed between our populations
but were hidden by the modularity of traits,
we re-created hybrids through time using multiple-point crossover recombination
rather than two-point crossover recombination.
%
In multiple-point crossover recombination,
each locus of a hybrid's genome had an equal probability
of coming from either parent;
in this way, modular traits could be broken apart by recombination.
%
We found that the strength of reproductive isolation
decreased for both ecological and mutation-order speciation,
such that mutation-order speciation
was almost as strong as ecological speciation (Figure \ref{hybrid_fitness}).
%
We even see some reproductive isolation in the drift treatment,
showing that incompatibilities also formed,
but at a much slower rate than speciation by natural selection.
%
These findings show that genetic incompatibilities
were hidden by the modularity of traits.
%
In other words, genetic incompatibilities that formed between populations
were not always seen in hybrids because two-point crossover recombination
did not break apart co-adapted gene complexes coding for a task.
%
We note that the genetic architecture of our populations
evolved under two-point crossover recombination,
not multiple-point crossover recombination,
and thus the modularity of traits
and formation of genetic incompatibilities
may be different under a different recombination method.



% intermediate environments and intermediate hybrids

% how much intrinsic?
% if parental halves could perform certain tasks,
%   but full hybrids could not, then intrinsic (at least partially)
%   must have occurred
%   - did it, found more intrinsic under divergent selection
%     - actual results, format: mean, (2.5%, 97.5%) bootstrap CI, 10000 reps
%       - MO-1: 69.68075, (40.73553, 101.78947)
%       - MO-2: 63.71429, (41.00000,  87.14286)
%       - Sel:  259.1875, (201.0000, 315.0047)
%     - so about 25.9% of cases were intrinsic for selection,
%       and about 6.6% of cases were intrinsic for mut-order
%     - better results, which shows proportion of intrinsic
%       relative to the total number of non-perfect hybrids
%       - MO-1: 0.220485, (0.1272438, 0.3289510)
%       - MO-2: 0.3458778, (0.2643781, 0.4205859)
%       - Sel:  0.2759421, (0.2144056, 0.3350845)
%     - so when considering only hybrids that do poorly,
%       the amounts of instrinsic among treatments is about the same
%   - method is similar logic by Dettman
% Rundle & Nosil ("Ecological speciation"):
%   it is possible that "genetic incompatibilities
%   evolve more rapidly under divergent selection
%   and that they are thus an important cause of ecological speciation"

%We found that in both treatments, mutation-order and ecological,
%the proportion of hybrids which were unfit
%due to (at least partially) intrinsic incompatibilities was about 30\%.
%%
%Thus, genetic incompatibilities did not evolve more frequently
%under divergent selection, as \citep{run05} predicted.

% the fact that little intrinsic formed in drift,
% implies that ecol. and mut-order spp promotes intrinsic

Part of the reason that hybrids were more unfit in the ecological treatment
than in the mutation-order treatment was that in the ecological treatment
more genetic incompatibilities (DMIs) formed between populations.
%
This result confirms the view that genetic incompatibilities
are a major cause of ecological speciation \citep{run05}.
%
Another reason that hybrids were more unfit in the ecological treatment was
that for a hybrid to be fully fit it had to inherit both sets of tasks from
both parents (i.e., four tasks), whereas for the mutation-order treatment,
hybrids required only two tasks to be fit.
%
In the ecological treatment, most hybrids inherited either one one two tasks
from one parent and none from the other, and in the mutation-order treatments,
most hybrids inherited both tasks from one parent, although in the second
mutation-order treatment hybrids often inherited one task from each parent.



The strength of selection in the low dimensionality environments
was greater than that in the high dimensionality environments.
%
An opposite trend may have made it difficult to know whether
it was higher dimensionality or stronger selection that resulted
in hybrids being less fit under high dimensionality than low dimensionality.
%
The weaker selection in high dimensionality may seem puzzling at first.
%
But given that each additional task an organism could perform gives it
an equal amount of merit, the higher the merit, the less an additional task
contributes to the total merit.
%
Therefore, the more tasks organisms can perform in the high dimensionality
environment, the less beneficial each one becomes (i.e., diminishing returns).
%
The strengths of selection in either environment are nevertheless high overall,
but it is not uncommon for selection to be high in new environments
\citep[e.g.,][]{len91,det07}.
%
Future studies could determine how the strength of selection may affect
the strength of postzygotic isolation
by testing a range of selection strengths.



In summary, we used the artificial life platform Avida, which allowed us to
precisely control the type of selection (divergent or uniform), to compare the
strength of reproductive isolation between ecological and mutation-order
speciation.
%
By accurately measuring the fitness of hybrids between populations, we showed
that ecological speciation formed stronger postzygotic isolation than
mutation-order speciation, although they were not so different when
recombination involved crossover at multiple points.
%
In addition, Avida allowed us to test various specific migration rates during
the evolution of population pairs, where we found that mutation-order
speciation was more sensitive to gene flow than ecological speciation.
%
We were also able to control the number of selection pressures in each
population, and we found that environments with high dimensionality formed
stronger reproductive isolation than those with low dimensionality.
%
These results support ideas brought up in the literature but which have been
difficult to test in biological organisms.
%
Avida provided a platform for us to test these ideas much more easily, and
although digital organisms are more simplistic than biological organisms, they
are a genetic system that evolves and speciates and therefore allows us to test
the generality of hypotheses about speciation, which often do not require the
specific details about how biological organisms work.



% Interestion question for discussion / future work
% is there a time-point in M-O in which mutations
% are no longer adaptive in BOTH populations due to the fact
% that early divergence causes divergent evolutionary paths?
% (idea from Schluter 2009)


% Simulations by Thibert find lower fitness
% in hybrids the stronger environmental diffs are.

% Computer simulations by Thibert-Plante find that increasing migration
% results in lower hybrid f5itness b/c hybrids compete with each other.
% We see the oppsoite---resources are not limited

% Dispersal had little effect on Fst (Thibert)



%Understanding mechanisms of speciation
%- m-o least understood
%  - may be important in small regions (similar environments), like islands
%  - but islands may have the most gene flow
%%- drift speciation is difficult (see Ch. 11 of Coyne & Orr)
%%  - very slow, especially in large populations
%%  - but may be important in founder events, bottlenecks
%% ** maybe put drift spp earlier



%gene flow constrains adaptive divergence during ecol. spp
%- we also see it (more) in mut-order

%Dimensionality in Avida is not necessarily orthogonal,
%as tasks may overlap in instruction use

% sch09b says that RI is easier by ecol. spp
% also: "There is a higher
% probability that the same or equivalent mutations will fix under
% parallel selection than under divergent selection, which will slow
% the rate of divergence between populations experiencing similar
% selection pressures"

% say somewhere that there is standing genetic variation,
% and that this contributed to parallel evolution in M.O.


%Luke: genetic distances, it's more than just that                               
%- call it something else, or use a real genetic distance                        
%- no. pairwise differences within pops and b/w pops,
%  1 - 1/bw 
%  - to correct for saturation: Nei's distance
%
%Include more on speciation in Discussion



\section{Acknowledgments}

I would like to thank my co-author on this work, Luke Harmon,
who reviewed this manuscript several times and provided me with great ideas.
%
I would also like to thank B. L. Williams
and the Digital Evolution Laboratory at MSU
for discussion and suggestions for improvement.
%
I am grateful to S. Singhal and two anonymous reviewers for their comments
that helped improve the quality of the manuscript.
%
This material is based in part upon work supported by
the National Science Foundation under Cooperative Agreement No. DBI-0939454.
%
Any opinions, findings, and conclusions or recommendations
expressed in this material are those of the author(s)
and do not necessarily reflect the views of the National Science Foundation.

\end{doublespace}

\bibliographystyle{apalike}
\bibliography{geo_ecol_spp}
